webdriver/bidi/network.json

```diff
@@ -101,6 +193,280 @@
+              "support": {
+                "chrome": {
+                  "version_added": "133",
+                  "partial_implementation": true,
+                  "notes": "Basic support"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "122"
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "request_parameter": {
+            "__compat": {
+              "description": "`request` parameter",
+              "spec_url": "https://w3c.github.io/webdriver-bidi/#cddl-key-networkbaseparameters-request",
+              "support": {
+                "chrome": {
+                  "version_added": "133",
+                  "partial_implementation": true,
+                  "notes": "Basic support"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "122"
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "response_parameter": {
+            "__compat": {
+              "description": "`response` parameter",
+              "spec_url": "https://w3c.github.io/webdriver-bidi/#cddl-key-networkauthrequiredparameters-response",
+              "support": {
+                "chrome": {
+                  "version_added": "133",
+                  "partial_implementation": true,
+                  "notes": "Basic support"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "122",
+                  "partial_implementation": true,
+                  "notes": "Basic support"
```

Comment:
@whimboo Would be good to add details about Firefox's partial implementation of the `response` parameter of the `network.authRequired` event, ideally linking to a bug.

Reply:
@juliandescottes could you please have a look?